easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-3.0.0_external_paraview-4.4.0.patch

```diff
@@ -0,0 +1,34 @@
+#Use external Paraview installed in $EBROOTPARAVIEW directory
+# B. Hajgato Nov 16 2015
+--- OpenFOAM-3.0.0/etc/config/paraview.csh.orig	2015-11-03 11:30:23.000000000 +0100
++++ OpenFOAM-3.0.0/etc/config/paraview.csh	2015-11-16 11:45:26.404431656 +0100
+@@ -87,7 +87,7 @@
+ set paraviewInstDir=$WM_THIRD_PARTY_DIR/ParaView-${ParaView_VERSION}
+ set paraviewArchName=ParaView-$ParaView_VERSION
+ 
+-setenv ParaView_DIR $WM_THIRD_PARTY_DIR/platforms/$WM_ARCH$WM_COMPILER/$paraviewArchName
++setenv ParaView_DIR $EBROOTPARAVIEW
+ 
+ # set paths if binaries or source are present
+ if ( -r $ParaView_DIR || -r $paraviewInstDir ) then
+--- OpenFOAM-3.0.0/etc/config/paraview.sh.orig	2015-11-03 11:30:23.000000000 +0100
++++ OpenFOAM-3.0.0/etc/config/paraview.sh	2015-11-16 11:46:07.296387935 +0100
+@@ -95,7 +95,7 @@
+ paraviewInstDir=$WM_THIRD_PARTY_DIR/ParaView-$ParaView_VERSION
+ paraviewArchName=ParaView-$ParaView_VERSION
+ 
+-export ParaView_DIR=$WM_THIRD_PARTY_DIR/platforms/$WM_ARCH$WM_COMPILER/$paraviewArchName
++export ParaView_DIR=$EBROOTPARAVIEW
+ 
+ # set paths if binaries or source are present
+ if [ -r $ParaView_DIR -o -r $paraviewInstDir ]
+--- OpenFOAM-3.0.0/applications/utilities/postProcessing/graphics/PV4Readers/Allwmake.orig	2015-11-03 11:30:23.000000000 +0100
++++ OpenFOAM-3.0.0/applications/utilities/postProcessing/graphics/PV4Readers/Allwmake	2015-11-16 11:47:06.130326063 +0100
+@@ -12,7 +12,6 @@
+         }
+ 
+         # ensure CMake gets the correct C++ compiler
+-        [ -n "$WM_CXX" ] && export CXX="$WM_CXX"
```

Comment:
is this a warning flag what we should be setting `$WM_CXX` in the OpenFOAM easyblock?